### DISCUSSION TITLE
laravel-envicon

### DISCUSSION THREAD

**Discussion by nhedger** [HassanZahirnia/laravel-package-ocean#105] [Package Suggestions]
I'd like to suggest a package of my own that I recently updated to work with Laravel 10.

https://github.com/nhedger/laravel-envicon

```
composer require hedger/laravel-envicon
```

**Comment by HassanZahirnia** [comment]
Hey Nicolas! 👋
Thanks for the suggestion 😊 I've added the package!

**Reply by nhedger** [reply]
Much appreciated @HassanZahirnia !
